### DISCUSSION TITLE
[ FF8 Steam ] Game crashes on opening save folder

### DISCUSSION THREAD

**Discussion by Chaos614** [julianxhokaxhiu/FFNx#435] [Q&A] [answered]
Anytime I have started the Steam version of FFVIII, the game will work just fine until I go to load or save the game, in which case it crashes mid load of the slot, I have added both the log as well as a picture and video of the issue in question

[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/8457631/FFNx.log)

![image](https://user-images.githubusercontent.com/31560086/162588283-b44d8aae-5858-4977-993c-6cb14d94956a.png)

https://user-images.githubusercontent.com/31560086/162588687-58ffa3b1-935d-4f08-8f97-8605ea72d5ec.mp4

Admittedly I played around in the menu's a bit to show the game worked fine otherwise without having to start a full New Game, but can add that too if need be

GPU is a Nvidia GeForce GTX 970
Driver Version 30.0.14.9649

All I know otherwise is that it was a fresh install, no mods or anything prior to this

**Comment by julianxhokaxhiu** [comment, marked as answer]
Hello, I just tested the game downloading it right now from Steam and applying the FFNx Steam edition 1.11 and it works just fine. I think you either have a corrupted installation or you have corrupted save files. Either way, double check your disk files and try again.

**Reply by Chaos614** [reply]
Ah, I double checked the save files and just deleted them all, and that seems to have done the trick, thank you!
